### ISSUE TITLE
Mod changelogs are unavailable until the mod is updated

### ISSUE THREAD

**Issue by rfyuz** [geode-sdk/geode#1091]
### Geode Issue

- [X] I confirm that this bug is NOT related to a mod but directly to Geode Loader itself.

### Platform

Android64

### Geode Version

v3.7.1

### Mods Installed

Irrelevant

### Expected Behavior

The purpose of changelogs are to see the changes associated with updating mods. Forcing the player to update before the changelogs can be viewed makes this complicated and makes determining whether you want to update a mod difficult.

I would expect the changelogs for all released versions, including those ahead of the installed version, of the installed mod to be available in the GUI, and perhaps an icon indicating which version you are currently on.

### Actual Behavior

Only the changelog up until the installed version is available

### Steps to Reproduce

1) Install any mod
2) Wait for an update to be available (or install a version of the mod that is not the latest version)
3) Check the changelogs and notice the changelog for the update is unavailable.

### Additional Information

Sorry if this is incorrectly labelled as a bug report; it feels like a bug to me.

**Comment by STGamer24YT** [comment]
Yeah that's very annoying
You could've posted this in the [discussions section](https://github.com/geode-sdk/geode/discussions) so it doesn't appear in the [issues section](https://github.com/geode-sdk/geode/issues)

**Comment by matcool** [comment]
> Yeah that's very annoying You could've posted this in the [discussions section](https://github.com/geode-sdk/geode/discussions) so it doesn't appear in the [issues section](https://github.com/geode-sdk/geode/issues)

this is an issue.

**Comment by Hanifinio** [comment]
I always found this annoying but thought it was intentional. New I'm thinking why would it be intentional in the first place

**Comment by SorkoPiko** [comment]
I agree. I never remember what mods I've updated after restarting so I never check the changelog.

**Comment by Hanifinio** [comment]
I always check where the my mods will appear in the Newest tab with the latest updated filter toggled. But would be more helpful to have this or have the filters also available in tab with the mods we have
